src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -183,7 +211,7 @@ case class WorkflowActor(workflow: WorkflowDescriptor,
 
   when(WorkflowRunning) {
     case Event(CallStarted(callKey), NoFailureMessage) =>
-      persistStatus(callKey, ExecutionStatus.Running)
+      Await.result(persistStatus(callKey, ExecutionStatus.Running), Duration.Inf)
```

Comment:
It looks like I added a bunch of `Await.result`s here but effectively there should actually be fewer.  There used to be an `Await.result` in `persistStatus`, but now that method doesn't block and now returns `Future[Unit]` instead of `Unit`.  Baby steps toward DSDEEPB-1549.